ProcessLib/LIE/HydroMechanics/LocalAssembler/HydroMechanicsLocalAssemblerFracture-impl.h

```diff
@@ -355,9 +363,9 @@ void HydroMechanicsLocalAssemblerFracture<ShapeFunctionDisplacement,
         b_m = ip_data.aperture0 + w[index_normal];
         if (b_m < 0.0)
         {
-            OGS_FATAL(
-                "Element %d, gp %d: Fracture aperture is %g, but it must be "
-                "non-negative.",
+            DBUG(
+                "Element %d, gp %d: Fracture aperture is %g, but it is "
+                "expected to be non-negative.",
                 _element.getID(), ip, b_m);
         }
```

Comment:
Forgot to set b_m = 0.0;

Reply:
Not sure here, because it is only used in the output, so if the solution is with negative aperture, so let it be visible.
In the assembly above, it is set to zero explicitly to avoid non-admissible range.
I'd like to keep the output as it is (allowing negative values).

Reply:
I see.